### ISSUE TITLE
onClick left or right of the image, image not changing properly

### ISSUE THREAD

**Issue by ardhendu12345** [yuvraj24/react-native-stories-view#41]
my code is
<StoryContainer
   visible={true}
   enableProgress={true}
   progressIndex={0}
   barStyle={{
   
    barWidth: 100, 
    barHeight: 4
 }}
   images={images}
   duration={20}  
   containerStyle={{
       width: '100%',
       height: '100%',
   }}
/>

**Comment by EmpiteRananjaya** [comment]
same issue here. any fix ?

**Comment by ardhendu12345** [comment]
> same issue here. any fix ?

no,

**Comment by Abhishek25031991** [comment]
how you have passed images array in storyContainer

**Comment by ardhendu12345** [comment]
i passed an array of images here

**Comment by Abhishek25031991** [comment]
There is a progressive bar issue as well. Most of the time the progressive bar fluctuate.

**Comment by ardhendu12345** [comment]
no its not fluctuate,  but  onLeft or Right click image is not getting changed... any solution?

**Comment by Abhishek25031991** [comment]
yes, I have solved this issue
 You need to work inside library to and increase the height of Touchability in ArrowNavi
<img width="1440" alt="Screenshot 2022-06-20 at 12 39 57 PM" src="https://user-images.githubusercontent.com/105962635/174544942-9cb16c65-5a07-43b9-8bf3-a7d70b6ba2f8.png">
gator file

**Comment by ardhendu12345** [comment]
its not working properly ...please help

**Comment by Abhishek25031991** [comment]
share your email id

**Comment by ardhendu12345** [comment]
ardhendugatait629@gmail.com

**Comment by R0LLeX** [comment]
Please tell me, can I transmit an array of images locally?

**Comment by Abhishek25031991** [comment]
No, They have used uri in the library, so you need to keep server images in array

**Comment by R0LLeX** [comment]
Okay, thank you very much for your help.

пн, 27 июн. 2022 г. в 07:35, Abhishek Pal ***@***.***>:

**Comment by ardhendu12345** [comment]
have any solution for this ...if yes please help

**Comment by Abhishek25031991** [comment]
share me your anydesk id and mobile number on email. I will help you out

**Comment by ardhendu12345** [comment]
Please help me ...

**Comment by yuvraj24** [comment]
Hey @Abhishek25031991 can you raise a PR of your change.....would be happy to review and merge the same !!!
